src/components/HorizontalCard.js

```diff
@@ -34,10 +22,17 @@ const Title = styled.p`
   font-size: 20px;
 `
 
-const HorizontalCard = ({ emoji, title, description, children, className }) => {
+const HorizontalCard = ({
+  emoji,
+  title,
+  description,
+  children,
+  className,
+  size,
+}) => {
   return (
     <StyledCard className={className}>
-      <Emoji svg text={emoji} className="horizontal-card-emoji" />
```

Comment:
We should delete the CSS selectors of this class (`.horizontal-card-emoji`) - I see it in wallets/index.js.